src/spryker/helm/app/templates/application/nginx/ingress.yaml

```diff
@@ -5,18 +5,18 @@ kind: Ingress
 metadata:
   creationTimestamp: null
   labels:
-    app.service: {{ .Values.prefix }}nginx
-  name: {{ .Values.prefix }}nginx
+    app.service: {{ .Values.resourcePrefix }}nginx
+  name: {{ .Values.resourcePrefix }}nginx
 spec:
   rules:
-  {{- range $key, $value := .Values.docker.services.nginx.environment }}
-  - host: {{ $value | quote }}
+  {{ range $key, $value := .Values.docker.services.nginx.environment }}
+  - host: "{{ $value }}"
     http:
       paths:
       - backend:
-          serviceName: {{ $root.Values.prefix }}nginx
+          serviceName: {{ $root.Values.resourcePrefix }}nginx
           servicePort: 80
-  {{- end }}
+  {{ end }}
```

Comment:
original was fine

Reply:
_base nginx ingress has it missing where I think this got copied from in the end, will add to both